### ISSUE TITLE
Width of tables

### ISSUE THREAD

**Issue by Juergen-aus-Koeln** [LotarProject/dokuwiki-template-bootstrap3#31]
Hi Lotar,

the width of tables is not always optimal. In this example the rows are very small, and I don't know the reason:
```
| Anleitung ^  =>  | {{:symbole:alg:benutzer.jpg?25&nolink}}    FAK                   | {{:symbole:sap:klasse.jpg?25&nolink}}          -                  |
|           | :::  | {{:symbole:sap:dokument.jpg?25&nolink}}    ASD / ANL             | {{:symbole:sap:objektzuordnung.jpg?25&nolink}} ggf. Materialstamm |
| :::       | :::  | {{:symbole:sap:dateiablage.jpg?25&nolink}} [Funktion] [MaschTyp] | {{:symbole:alg:dateilink.jpg?25&nolink}}       -                  |
```
Curiously in the resulting table there is a line break after each picture. Other tables are working correct.

The option "tableFullWidth" in the admin panel is off.

Regards Juergen

**Comment by LotarProject** [comment]
Hi Juergen. I have created the page with your code and this is the result:

Default Dokuwiki theme:
![image](https://cloud.githubusercontent.com/assets/1824717/7682346/ba4acd4a-fd77-11e4-9b17-56d0d24a381a.png)

Bootstrap3 theme:
![image](https://cloud.githubusercontent.com/assets/1824717/7682320/8f8fc1a0-fd77-11e4-9d44-57ffd7824187.png)

This is the expected result?

Best regards,
Joseph

**Comment by Juergen-aus-Koeln** [comment]
Yes, this would be the expected result. But it looks different. I can't include an image into my text, but I can give you the html code. Perhaps you can find something remarkable:
```
<h2 class="sectionedit2" id="parameter">Parameter</h2>
<div class="level2">
<div style="width: auto;" class="table sectionedit3 table-responsive"><table style="width: auto;" class="inline table table-striped table-condensed">
	<tbody><tr class="row0">
		<td class="col0"> Anleitung </td><th class="col1 centeralign" rowspan="3">  ⇒  </th><td class="col2 leftalign"> <img src="/dokuwiki/lib/exe/fetch.php?w=25&amp;tok=d43afe&amp;media=symbole:alg:benutzer.jpg" class="media img-responsive" title="benutzer.jpg" alt="benutzer.jpg" width="25">    <abbr title="Fachabteilungen Konstruktion">FAK</abbr>                   </td><td class="col3 leftalign"> <img src="/dokuwiki/lib/exe/fetch.php?w=25&amp;tok=60ca49&amp;media=symbole:sap:klasse.jpg" class="media img-responsive" title="klasse.jpg" alt="klasse.jpg" width="25">          -                  </td>
	</tr>
	<tr class="row1">
		<td class="col0 leftalign" rowspan="2">           </td><td class="col1 leftalign"> <img src="/dokuwiki/lib/exe/fetch.php?w=25&amp;tok=95a55a&amp;media=symbole:sap:dokument.jpg" class="media img-responsive" title="dokument.jpg" alt="dokument.jpg" width="25">    ASD / ANL             </td><td class="col2"> <img src="/dokuwiki/lib/exe/fetch.php?w=25&amp;tok=8175a1&amp;media=symbole:sap:objektzuordnung.jpg" class="media img-responsive" title="objektzuordnung.jpg" alt="objektzuordnung.jpg" width="25"> ggf. Materialstamm </td>
	</tr>
	<tr class="row2">
		<td class="col0"> <img src="/dokuwiki/lib/exe/fetch.php?w=25&amp;tok=e47908&amp;media=symbole:sap:dateiablage.jpg" class="media img-responsive" title="dateiablage.jpg" alt="dateiablage.jpg" width="25"> [Funktion] [MaschTyp] </td><td class="col1 leftalign"> <img src="/dokuwiki/lib/exe/fetch.php?w=25&amp;tok=c2e0d7&amp;media=symbole:alg:dateilink.jpg" class="media img-responsive" title="dateilink.jpg" alt="dateilink.jpg" width="25">       -                  </td>
	</tr>
</tbody></table></div>
```
Best regards
Juergen

**Comment by LotarProject** [comment]
I dont see particular difference with my html.
Is possible to send me a screenshot of the table ?

Best regards
Joseph

**Comment by Juergen-aus-Koeln** [comment]
I've send you the screenshot by mail

I've tested it with several browsers, the effect is the same everywhere.

Best regards
Juergen

**Comment by LotarProject** [comment]
Hi. I have found the problem. Few days ago i have committed (4479d1d) a fix to correct the image alignment.
I have discovered the ".img-responsive" Bootstrap class, add for all "<img>" the CSS property "display:block". This property cause the line break after the image. The fix, change the "display:block" in "display:inline-block".

I suggest to download the new version of template and/or clear the DokuWiki cache (via URL "/dokuwiki/lib/exe/css.php?purge=true").

Best regards,
Joseph
